src/main/scala/com/workday/warp/math/linalg/AnomalyDetector.scala

```diff
@@ -63,8 +78,28 @@ case class RobustPcaRunner(lPenalty: Double = WARP_ANOMALY_RPCA_L_PENALTY.value.
       rawResponseTimes
     }
 
+    val dickeyFuller: AugmentedDickeyFuller = new AugmentedDickeyFuller(truncatedData.toArray)
+
+    val trend: Double = dickeyFuller.zeroPaddedDiff.sum
+
+    // we want a tight threshold
+    val testedData: Iterable[Double] = this.useDiff match {
+      // check user overrides
+      case Some(true) =>
+        Logger.debug(s"override in place (useDiff=true). time series will be treated as non-stationary.")
+        dickeyFuller.zeroPaddedDiff
+      case Some(false) =>
+        Logger.debug(s"override in place (useDiff=false). time series will be treated as stationary.")
+        truncatedData
+      // check actual results of the test
+      case _ if !dickeyFuller.isStationary && trend < 0.0 =>
```

Comment:
this `trend < 0.0` check is kind of hacky, but i noticed that with a global increasing trend, our smart threshold will closely follow that trend, ie a sustained sequence of anomalous data points will quickly be considered normal

Reply:
does sensitivity factor play a role in influencing whether a set of data points can be considered anomalous? Wondering if lowering/increasing the sensitivity factor would help mitigate some of that

Reply:
hmm i think sensitivity factor will shift the smart threshold up or down, but won't affect the jittering aspect. ill double check.